### DISCUSSION TITLE
Missing Russian language

### DISCUSSION THREAD

**Discussion by AaronHuston** [outline/outline#5706] [General] [answered]
After the update, the Russian language disappeared in the interface. Intentional deletion?

**Comment by tommoor** [comment, marked as answer]
Hi, yes this was intentional - we’re no longer supporting Russian language going forward.

**Reply by snarkyalyx** [reply]
Is there a reason to that, or is it just because the lack of a maintainer?

**Reply by tommoor** [reply]
Russia [no longer respects western copyright, trademarks, or intellectual property rights](https://www.lexology.com/library/detail.aspx?g=00f5d29e-50ea-4449-a56e-9693b12e905e). Several Russian companies have stolen the Outline codebase. As such I'm removing Russian language support, no need to make it easier than it needs to be.

**Reply by inkloop** [reply]
russia is a terrorist state, what else do you need to know?
